automation_tools/__init__.py

```diff
@@ -373,41 +373,36 @@ def setup_default_capsule(interface=None, run_katello_installer=True):
             print('Was not possible to fetch interface information')
             sys.exit(1)
 
-    if os.environ.get('SATELLITE_VERSION') in ('6.0', '6.1'):
-        proxy = 'capsule'
-    else:
-        proxy = 'foreman-proxy'
-
     installer_options = {
-        '{0}-dns'.format(proxy): 'true',
-        '{0}-dns-forwarders'.format(proxy): forwarders,
-        '{0}-dns-interface'.format(proxy): interface,
-        '{0}-dns-zone'.format(proxy): domain,
-        '{0}-dhcp'.format(proxy): 'true',
-        '{0}-dhcp-interface'.format(proxy): interface,
-        '{0}-tftp'.format(proxy): 'true',
-        '{0}-tftp-servername'.format(proxy): hostname,
-        '{0}-register-in-foreman'.format(proxy): 'true',
+        'foreman-proxy-dns': 'true',
+        'foreman-proxy-dns-forwarders': forwarders,
+        'foreman-proxy-dns-interface': interface,
+        'foreman-proxy-dns-zone': domain,
+        'foreman-proxy-dhcp': 'true',
+        'foreman-proxy-dhcp-interface': interface,
+        'foreman-proxy-tftp': 'true',
+        'foreman-proxy-tftp-servername': hostname,
+        'foreman-proxy-register-in-foreman': 'true',
     }
 
     # Handles enabling puppet support
     if not os.environ.get('SATELLITE_VERSION') in ('6.3', '6.4'):
-        installer_options['{0}-puppetca'.format(proxy)] = 'true'
+        installer_options['foreman-proxy-puppetca'] = 'true'
         if os.environ.get('SATELLITE_VERSION') in ('6.0', '6.1', '6.2'):
```

Comment:
isn't it safe to reduce this to `6.2`?

Reply:
ok, wanted to maintain original context as it seems we miss to add ```'6.5'``` and ```'6.6'``` to ```('6.3', '6.4')```